todo_app/templates/index.html

```diff
@@ -12,7 +12,21 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for task in taskList %}
+        <li>{{ task.id }}:  {{ task.status }}:  {{ task.title }}</li>
+        {% endfor %}
       </ul>
+
+      <form action ='/add' method='POST'>
+      
+      <fieldset>
+        <legend>Add Task</legend>
+        <input type="text" name='newTask'>
```

Comment:
Really minor, but adding a `required` attribute to this input would ensure that my browser stops me accidentally submitting an empty form - it's not enough to avoid people intentionally aiming to do so, but might improve the UX a little!